csfieldguide/package.json

```diff
@@ -13,11 +13,13 @@
     "sticky-state": "2.4.1",
     "iframe-resizer": "3.6.3",
     "csfg-module-fsa": "file:./static/js/modules/fsa/",
+    "csfg-interactive-algorithm-timer": "file:./static/interactives/algorithm-timer/",
```

Comment:
Various combinations of numbers at their highest values result in console errors
With n = 1000 there is no error but the result is still infinity

![image](https://user-images.githubusercontent.com/25916475/57048809-97a65f00-6cc9-11e9-9478-5f9a538c7235.png)

Reply:
Also perhaps the number of decimal places should be limited
![image](https://user-images.githubusercontent.com/25916475/57049027-932e7600-6cca-11e9-8f15-3856e2d88a18.png)